packages/jest-snapshot/src/__tests__/snapshot.js

```diff
@@ -0,0 +1,82 @@
+/**
```

Comment:
It's great that you wrote an integration test! However, we should definitely also write unit tests for individual pieces of the snapshot functionality. This specific test should probably go into the `integration_test` folder because it tries to make sure that the matchers actually work.

Also, it should never actually write to the real file system. You can see how the file-system is mocked here: https://github.com/facebook/jest/blob/master/packages/jest-haste-map/src/__tests__/worker-test.js#L29 – you can model your test after this. If you mock the fs functions used in snapshot testing, you can do proper unit testing without ever actually writing a file :)